cernopendata/modules/fixtures/data/terms/terms.json

```diff
@@ -12746,5 +12746,328 @@
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "DELPHI"
+      }
+    ],
+    "term": [
+      "CERNLIB"
+    ],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "LEP",
+    "category": "generic",
+    "definition": "The Large Electron-Positron Collider",
+    "links": [
+      {
+        "text": "LEP page at CERN",
+        "url": "https://home.cern/science/accelerators/large-electron-positron-collider"
+      },
+      {
+        "text": "The LEP story",
+        "url": "https://home.cern/news/press-release/cern/lep-story"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "LEP"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "ALEPH",
+    "category": "generic",
+    "definition": "ALEPH was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "ALEPH home page",
+        "url": "https://aleph.web.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "ALEPH"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "DELPHI",
+    "category": "generic",
+    "definition": "DEtector with Lepton, Photon and Hadron Identification. DELPHI was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "DELPHI home page",
+        "url": "https://delphi-www.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "L3"
+      },
+      {
+        "term": "OPAL"
+      }
+    ],
+    "term": [
+      "DELPHI"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "OPAL",
+    "category": "generic",
+    "definition": "Omni-Purpose Apparatus at LEP. OPAL was one out of four experiments of the LEP collider",
+    "links": [
+      {
+        "text": "OPAL home page",
+        "url": "https://opal.web.cern.ch"
+      }
+    ],
+    "see_also": [
+      {
+        "term": "LEP"
+      },
+      {
+        "term": "ALEPH"
+      },
+      {
+        "term": "DELPHI"
+      },
+      {
+        "term": "L3"
+      }
+    ],
+    "term": [
+      "OPAL"
+    ],
+    "experiment": ["ALEPH", "DELPHI", "L3", "OPAL"],
+    "type": {
+      "primary": "Glossary"
+    }
+  },
+  {
+    "anchor": "L3",
+    "category": "generic",
+    "definition": "L3 was one out of four experiments of the LEP collider",
```

Comment:
Is L3 an acronym? If so, it would be nice to explain it (for the record, I did look at https://home.cern/science/experiments/l3 , and I couldn't find out why it was called L3 :( )